### ISSUE TITLE
Accessibility: Login screen not accessible

### ISSUE THREAD

**Issue by shortwavesurfer2009** [AgoraDesk-LocalMonero/agoradesk-app-foss#246]
When using talkback the username, password, and OTP boxes are not able to be opened.
